glibc-217/Dockerfile

```diff
@@ -7,19 +7,27 @@ RUN groupadd --force --gid $GROUP_ID node
 RUN adduser --gid $GROUP_ID --uid $USER_ID node
 
 RUN ulimit -n 1024 \
+    && sed -i -e 's!mirrorlist!#mirrorlist!g' /etc/yum.repos.d/CentOS-* \
+    && sed -i -e 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-* \
     && yum install -y epel-release \
     && yum install -y centos-release-scl-rh \
+    && sed -i -e 's!mirrorlist!#mirrorlist!g' /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo \
+    && if [ "$(uname -m)" == "x86_64" ]; then \
+      sed -i -e 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo; \
+    else \
+      sed -i -e 's|#baseurl=http://mirror.centos.org/centos|baseurl=http://vault.centos.org/altarch|g' /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo; \
+    fi \
     && yum upgrade -y \
     && yum install -y \
-         git \
-         curl \
-         make \
-         python2 \
-         python3 \
-         ccache \
-         xz-utils \
-         devtoolset-9 \
-         glibc-devel
+        git \
+        curl \
+        make \
+        python2 \
+        python3 \
+        ccache \
+        xz \
```

Comment:
why this change?

Reply:
ups good catch. That was just legacy from testing possible solutions until we get into this state. Just removed those changes @jbudz